docs/en/latest/debug-mode.md

```diff
@@ -37,6 +37,7 @@ You can enable the basic debug mode by adding this line to your debug configurat
 basic:
   enable: true
 ```
+APISIX syncs the configuration passed in debug.yaml on startup and then checks if the file was modified on an interval of 1 second. If the file is changed, then it resyncs the debug configuration so that users don't have to reload for the debug configuration to take effect.
```

Comment:
```suggestion
APISIX loads the configurations of `debug.yaml` on startup and then checks if the file is modified on an interval of 1 second. If the file is changed, APISIX automatically applies the configuration changes.

Reply:
@Revolyssup could you help confirm in code if:
* 1 second is the default
* 1 second could be modified by config under `/conf` (or is it hardcoded)

Reply:
@kayx23  Here you can see its hardcoded https://github.com/apache/apisix/blob/01c583d24919e192085fda40a5e247e2a406d715/apisix/debug.lua#L311